packages/core/contracts/financial-templates/contract-for-difference/ContractForDiffrenceCreator.sol

```diff
@@ -65,26 +65,25 @@ contract ContractForDifferenceCreator is Testable, Lockable {
         bytes32 priceIdentifier,
         string memory syntheticName,
         string memory syntheticSymbol,
-        address collateralAddress,
-        address financialProductLibraryAddress,
+        IERC20Standard collateralToken,
+        address financialProductLibrary,
```

Comment:
Can we pass this in as a `ContractForDifferenceFinancialProductLibrary`?

Reply:
yup! I also made the same change to the finder. we have a local finder instance so no need to cast it!